Supporting conda install in kedro install command (#33)
quantumblacklabs/kedro: docs/source/02_getting_started/03_new_project.md

```

This command will install dependencies listed in `src/requirements.txt`.

> If your project has conda dependencies to install use the following:

Reviewer: ```suggestion
If your project has conda dependencies which you would like to install with this command, you can create a `src/environment.yml` file and list them there.
```

> ```bash

Reviewer: ```suggestion
```

> kedro install --conda

Reviewer: ```suggestion
```

> ```

Reviewer: ```suggestion
```

> This command will install dependencies listed in `src/environment.yml`. 

Reviewer: ```suggestion

```

> Please use the template provided to base your `enviroment.yml` off.  

Reviewer: ```suggestion
```
